SalesforceCommerceCloud/pwa-kit
# Checkout should default to Guest Shopper

**GUS**: (https://gus.my.salesforce.com/a07AH000000OOhDYAW)


## How to test-drive this PR
- Checkout and run
- Navigate as guest to the checkout
- Confirm that the password field is not showing & Continue as guest is default as per ticket
